test/smoke/src/main.ts

```diff
@@ -34,7 +34,7 @@ import { setup as setupDataMultirootTests } from './areas/multiroot/multiroot.te
 import { setup as setupDataLocalizationTests } from './areas/workbench/localization.test';
 import { setup as setupLaunchTests } from './areas/workbench/launch.test';
 
-const tmpDir = tmp.dirSync({ prefix: 't' }) as { name: string; removeCallback: Function; };
+const tmpDir = tmp.dirSync({ name: 't' }) as { name: string; removeCallback: Function; };
```

Comment:
Smoke tests in ci were creating socket paths longer than we expect https://github.com/microsoft/vscode/blob/5fedf0213401d8b928024151aedd589a81d9cf78/src/vs/base/parts/ipc/node/ipc.net.ts#L522-L523 , which was causing the app to quit with exception, hence this change to reduce the length of user data dir.